icons/gocd.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>GoCD</title><path d="m18.043 13.237-8.908 5.936a1.47 1.47 0 0 1-.822.25 1.449 1.449 0 0 1-.696-.174 1.48 1.48 0 0 1-.784-1.308v-5.94a1.482 1.482 0 1 1 2.956 0v3.163l4.748-3.163-7.06-4.696A1.49 1.49 0 1 1 9.13 4.823l8.913 5.941a1.492 1.492 0 0 1 0 2.473M11.999 0a12 12 0 1 0 12 12A12.012 12.012 0 0 0 12 0"/></svg>
```

Comment:
Near perfect, the only differences I see is the following:
![image](https://user-images.githubusercontent.com/12817388/130072628-6c7885ec-364c-4d86-aa40-71efe6ca0e0f.png)

It is probably from the optimization, you can try with a precision of 4 if it solves the difference.

Reply:
![image](https://user-images.githubusercontent.com/28590126/131330422-2b315160-ab2b-461e-acaf-ec95e82ca842.png)

It did not change anything. If you want me to do something more please let me know, however I don't think I can do much, since I'm not that good with svg-s.

Maybe you can help me if you still consider that's an issue.

Reply:
> It did not change anything.

Did you change the precision on the already optimized icon? If so, it wont change anything.
You might need to scale the original again and optimize it with a precision of 4.

> however I don't think I can do much, since I'm not that good with svg-s.

It was near perfect, so you're not that bad, trust me :)

Let me know if it does not work and I will give it a try.

Reply:
> Did you change the precision on the already optimized icon?

Unfortunately yes.

> Let me know if it does not work and I will give it a try.

I would appreciate if you could do it, cuz it took me hours to get it into the state it is now, and I am running out of time since I have some deadlines to meet, as I was in vacation for 3 weeks until 2 days ago. :( 

If you don't have time then tbh idk when I will have time to do it again.

Sorry and Thanks!

Reply:
No problem, you can try the following path (based on `gocd_circle_mark_filled_black.svg`):
```
M18.043 13.237l-8.907 5.935a1.47 1.47 0 01-.823.25 1.449 1.449 0 01-.696-.173 1.48 1.48 0 01-.784-1.308V12a1.482 1.482 0 112.957 0v3.163L14.537 12 7.478 7.304A1.49 1.49 0 119.13 4.823l8.913 5.94a1.492 1.492 0 010 2.474M12 0a12 12 0 1012 12A12.012 12.012 0 0012 0
```

If you use it, I will ask another @simple-icons/maintainers to review it

Reply:
Changed it with your suggestion.

Thanks a lot @service-paradis

Reply:
No problem :)
Since you used my path, I will ask another @simple-icons/maintainers to review it!

Reply:
> No problem :)
> Since you used my path, I will ask another @simple-icons/maintainers to review it!
![image](https://user-images.githubusercontent.com/15719479/132917719-98541035-21f1-4d9c-a20c-346cb718da5c.png)

100% match from me @service-paradis. LGTM!